### PR TITLE
Refresh widgets when URL parameters change

### PR DESCRIPTION
**Pull request by jmuharsky**

### CODE REVIEW

File: client/components/dashboard/dashboard-service.js [hunk 1]
```diff
@@ -182,6 +198,46 @@ DashboardService.prototype.saveDashboard = function() {
...
 
 /**
+ * Iterates through the dashboard and applies functions to the contents.
+ * @param {!DashboardConfig} dashboard The dashboard config to iterate.
+ * @param {?function(container)} updateContainerFn The function to apply to
+ *    each container.
+ * @param {?function(widget)} updateWidgetFn The function to apply to each
+ *    widget.
+ * @param updateWidgetFn
+ * @constructor
```
Comment (cmccoy) [inline client/components/dashboard/dashboard-service.js]: Is this a constructor?

File: client/components/dashboard/dashboard-service.js [hunk 2]
```diff
@@ -182,6 +198,46 @@ DashboardService.prototype.saveDashboard = function() {
...
+ */
+DashboardService.prototype.UpdateDashboard = function(
```
Comment (cmccoy) [inline client/components/dashboard/dashboard-service.js]: Nit: check function case.

**Comment by cmccoy** [comment]
Two little things, otherwise LGTM. Consider squashing these commits.

**Comment by jmuharsky** [comment]
Addressed and squashed.  PTAL?

File: client/components/dashboard/dashboard-service.js [hunk 3]
```diff
@@ -182,6 +198,45 @@ DashboardService.prototype.saveDashboard = function() {
...
 /**
+ * Iterates through the dashboard and applies functions to the contents.
+ * @param {!DashboardConfig} dashboard The dashboard config to iterate.
+ * @param {?Function(ContainerConfig)} updateContainerFn The function to apply
+ *    to each container.
+ * @param {?Function(WidgetConfig)} updateWidgetFn The function to apply to each
+ *    widget.
+ * @param updateWidgetFn
+ */
+DashboardService.prototype.UpdateDashboard = function(
```
Comment (cmccoy) [inline client/components/dashboard/dashboard-service.js]: Nit: check function case - should this be `updateDashboard`?

**Comment by cmccoy** [comment]
One comment again, otherwise LGTM.

**Pull request merged**